src/window/User Guide.md

```diff
@@ -0,0 +1,132 @@
+% User Guide to Using Piston Windows
+
+# Guide to using Piston Windows
+
+## Getting Help
+
+*Notice! Piston is being actively developed. Please let us know if anything in this tutorial is outdated.*
+
+See the main [Piston Guide](../../GUIDE.md) for general help with Piston.
+
+## Overview
+
+The piston engine is designed to be an abstraction. It provides the same functionality through several different backends, each of which have their own quirks and extra features.
+
+Because of this abstraction, some of the details of the piston API may seem unnecessary or useless. The purpose of this guide is to give concrete guidelines on the creation and usage of piston windows.
```

Comment:
Perhaps could add a disclaimer or a sentence that explains which audience. I tend to not like splitting people into categories like beginner/experienced, because people are often quick learners and making them think of themselves as beginners might slow them down. In psychology this is called ["priming"](https://en.wikipedia.org/wiki/Priming_%28psychology%29).

I picture this: A is what you want and B is what you do. Let's create an arrow `A -> B` straight from A to B. Just show it.